src/hotspot/share/include/jmm.h

```diff
@@ -80,7 +81,8 @@ typedef enum {
   JMM_COMPILE_TOTAL_TIME_MS          = 8,    /* Total accumulated time spent in compilation */
   JMM_GC_TIME_MS                     = 9,    /* Total accumulated time spent in collection */
   JMM_GC_COUNT                       = 10,   /* Total number of collections */
-  JMM_JVM_UPTIME_MS                  = 11,   /* The JVM uptime in milliseconds */
+  JMM_GC_CPU_TIME                    = 11,   /* Total accumulated GC CPU time */
+  JMM_JVM_UPTIME_MS                  = 12,   /* The JVM uptime in milliseconds */
```

Comment:
It looks a bit odd to me to change the existing define of UPTIME here.  
OK it is not a public interface used between different versions, and people should not be mixing up jmm.h and management implementations...  But usually we would just add the new definition? (items below are not strictly grouped by name)  Maybe this is just a nit, and only makes cross-version comparisons easier.

Looks good overall.